index_configuration_tool/main.py

```diff
@@ -19,10 +21,9 @@ def get_auth(input_data: dict) -> Optional[tuple]:
 
 
 def get_endpoint_info(plugin_config: dict) -> tuple:
-    endpoint = "https://" if ("ssl" in plugin_config and plugin_config["ssl"]) else "http://"
```

Comment:
This logic is gone because the endpoint url needs to be fully specified in the new format? or another reason?

Reply:
Yes, the Data Prepper pipeline configuration requires the full URI